modules/enabling-gpu-support-in-data-science.adoc

```diff
@@ -0,0 +1,30 @@
+:_module-type: PROCEDURE
+
+[id='enabling-gpu-support-in-data-science_{context}']
```

Comment:
Maybe just `enabling-gpu-support_{context}`? I must confess, I don't love 'data science' as a more general form of the product name to fit the upstream; it doesn't seem like a great match with 'Open Data Hub'. I'd sooner leave the product name out of the ID entirely (and hence out of the URL).